src/Catalog/Helpers/PackageUtility.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.Metadata.Catalog.Helpers
+{
+    public static class PackageUtility
+    {
+        public static string GetPackageFileNameLowercase(string packageId, string packageVersion)
```

Comment:
Is this package version or normalized version? If normalized it would be great to have this in the argument name.

Reply:
It's indeterminate.  The caller is responsible for normalizing.

Adding an option for normalizing would be a nice to have refactoring, I'll punt it from this PR.